### PR TITLE
Skipping searchbar preview on reload

### PR DESCRIPTION
**Pull request by jacob-nv**
## Description
Setting "search term" from route would trigger search, which would create request on page load.
Added additional bool value to check if this change is triggered by route or user input

## Related Issue

- Fixes https://github.com/owncloud/web/issues/10200

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by JammingBen — COMMENTED** [review]
LGTM and works as expected 👍 Could you please add a changelog item for the bugfix?

File: packages/web-app-search/src/portals/SearchBar.vue [hunk 1]
```diff
@@ -129,6 +129,7 @@ export default defineComponent({
     const optionsDropRef = ref(null)
     const activePreviewIndex = ref(null)
     const term = ref('')
+    const loadFromRoute = ref(false)
```
Comment (kulmann) [inline packages/web-app-search/src/portals/SearchBar.vue:132]: Small nitpick about the variable naming: I'd like to name this something like `restoreSearchFromRoute` to make it more descriptive.

Comment (jacob-nv) [reply packages/web-app-search/src/portals/SearchBar.vue]: Fixed

**Review by JammingBen — COMMENTED** [review]
LGTM and works as expected AFAICT 👍  Could you please add a changelog item?

**Review by kulmann — APPROVED** [review]
👍

**Review by JammingBen — APPROVED** [review]

**Pull request merged**